src/test/java/org/schemaspy/testcontainer/MSSQLServerIT.java

```diff
@@ -0,0 +1,124 @@
+
+    private static Database database;
+
+    @ClassRule
+    public static JdbcContainerRule<MSSQLServerContainer> jdbcContainerRule =
+            new JdbcContainerRule<>(() -> new MSSQLServerContainer())
+                    .assumeDockerIsPresent()
+                    .withAssumptions(assumeDriverIsPresent())
+                    .withInitScript("integrationTesting/dbScripts/mssql.sql");
+
+    @Before
+    public synchronized void gatheringSchemaDetailsTest() throws SQLException, IOException, ScriptException, URISyntaxException {
+        if (database == null) {
+            createDatabaseRepresentation();
+        }
+    }
+
+    private void createDatabaseRepresentation() throws SQLException, IOException, URISyntaxException {
+        String[] args = {
+                "-t", "mssql12",
+                "-db", "test",
+                "-s", "dbo",
+                "-cat", "%",
+                "-o", "target/integrationtesting/mssql",
+                "-u", "sa",
+                "-p", jdbcContainerRule.getContainer().getPassword(),
+                "-host", jdbcContainerRule.getContainer().getContainerIpAddress(),
+                "-port", jdbcContainerRule.getContainer().getMappedPort(1433).toString()
+        };
+        given(arguments.getOutputDirectory()).willReturn(new File("target/integrationtesting/databaseServiceIT"));
+        given(arguments.getDatabaseType()).willReturn("mssql12");
+        given(arguments.getUser()).willReturn("sa");
+        given(arguments.getSchema()).willReturn("dbo");
+        given(arguments.getCatalog()).willReturn("%");
+        given(arguments.getDatabaseName()).willReturn("test");
+        Config config = new Config(args);
+        DatabaseMetaData databaseMetaData = sqlService.connect(config);
+        Database database = new Database(config, databaseMetaData, arguments.getDatabaseName(), arguments.getCatalog(), arguments.getSchema(), null, progressListener);
+        databaseService.gatheringSchemaDetails(config, database, progressListener);
+        this.database = database;
+    }
+
+    @Test
+    public void databaseShouldBePopulatedWithTableTest() {
+        Table table = getTable("TestTable");
+        assertThat(table).isNotNull();
+    }
+
+    @Test
+    public void databaseShouldBePopulatedWithTableTestAndHaveColumnName() {
+        Table table = getTable("TestTable");
+        TableColumn column = table.getColumn("Description");
+        assertThat(column).isNotNull();
+    }
+
+    @Test
+    public void databaseShouldBePopulatedWithTableTestAndHaveColumnNameWithComment() {
+        Table table = getTable("TestTable");
+        TableColumn column = table.getColumn("Description");
+        assertThat(column.getComments()).isEqualToIgnoringCase("This is column description");
+    }
+
+    private Table getTable(String tableName) {
+        return database.getTables().stream().filter(table -> table.getName().equals(tableName)).findFirst().get();
```

Comment:
I somewhat feel ashamed, since it copy & past from mine.
But `return database.getTablesByName().get(tableName);` is probably better.